app/assets/stylesheets/master.css.scss

```diff
@@ -10,7 +19,8 @@
 .enter {
   font-family: "Special Elite";
   font-size: 45px;
-  color: #C9D1D4;
+  // color: #C9D1D4;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.